main.py

```diff
@@ -1,12 +1,226 @@
+                        id = input("Which video id would you want to select? ")
+                        if id.isnumeric():
+                            id = int(id)
+                            store_employee.get_a_video(id=id)
+                    else:
+                        print("Could not select. Please enter id or title")
 
+                    if store_employee.selected_video:
+                        print("selected video: ", store_employee.selected_video)
+                    if not store_employee.selected_video:
+                        print("Could not find a video by that title or id")    
+                elif option_chosen == "3":
+                    title =input("What is the title of your video? ")
+                    release_date =input("What is the video's release date? ")
+                    total_inventory =input("What is the video's total_inventory? ")
+                    response = store_employee.add_video(title=title, release_date=release_date, total_inventory=total_inventory)
+                    print("New video", response)
+                elif option_chosen == "4":
+                    print(f"Great! Let's update the video: {store_employee.selected_video}")
+                    title =input("What is the new title of your video? ")
+                    release_date =input("What is the new video's release date? ")
+                    total_inventory =input("What is the new video's total_inventory? ")
+                    response = store_employee.update_video(title=title, release_date=release_date, total_inventory=total_inventory)
+                    print(response)
+                elif option_chosen == "5":
+                    store_employee.delete_video()
+                    print("The video has been deleted")
+                elif option_chosen == "6":         
+                    exit = True           
+        elif choice == "3":
+            exit = False
+            while not exit:
+                print_stars()
+                rental_options = manage_rentals()
+                print_stars()
+                option_chosen= make_a_choice(rental_options)
+                if option_chosen == "1":
+                    customer_id = input("What is the customer id? ")
+                    video_id = input("What is the video id? ")
+                    response = store_employee.check_out(customer_id=customer_id, video_id=video_id)
+                    print(response)
+                elif option_chosen == "2":
+                    customer_id = input("What is the customer id? ")
+                    video_id = input("What is the video id? ")
+                    response = store_employee.check_in(customer_id=customer_id, video_id=video_id)
+                    print(response)
+                elif option_chosen == "3":
+                    exit = True
+        elif choice == "4":
+            print_stars()
+            print("Thanks for using our store!")
+            print_stars()  
+            quit = True
```

Comment:
Your logic looks great Priscille! A future refactor can include placing the customer , rental, and video options into different files.